src/containers/apply/ApplyForm.tsx

```diff
@@ -26,179 +26,148 @@ interface IApplyProps extends IApplication {
 }
 
 type ApplicationDispatch = ThunkDispatch<
-  IRootState,
-  undefined,
-  actions.SubmitFormAction | actions.UpdateApplicationAction
+IRootState,
+undefined,
+actions.SubmitFormAction | actions.UpdateApplicationAction
```

Comment:
These lines seem like they should be indented
```suggestion
  IRootState,
  undefined,
  actions.SubmitFormAction | actions.UpdateApplicationAction
```